src/lib/arrays.js

```diff
@@ -0,0 +1,89 @@
+const getNthElement = (index, array) => {
```

Comment:
Im going to go over the `/lib` files as have already reviewed them last time round!